CIP-onchain-cert-metadata/README.md

```diff
@@ -0,0 +1,489 @@
+    "logo": "https://www.example.com/media/logo.svg"
+    "social": {
+        "contact": "contact@example.com",
+        "link": "https://example.com",
+        "twitter": "twiterHandle",
+        "github": "githubHandle",
+        "website": "https://www.example.com"
+    }
+  },
+  "report": {
+    "reportURLs": [
+        "https://example.io/certificate.pdf",
+        "ipfs://bafybeiemxfal3jsgpdr4cjr3oz3evfyavhwq"
+        ],
+    "reportHash": "c6bb42780a9c57a54220c856c1e947539bd15eeddfcbf3c0ddd6230e53db5fdd"
+  },
+  "summary": "This is the summary of the report."
+  "disclaimer": "This is the legal disclaimer from the report",
+  "scripts": [
+        {
+          "smartContractInfo": {
+            "era": "basho",
+            "compiler": "plutusTx",
+            "compilerVersion": "1.3.0",
+            "optimizer": "plutonomy",
+            "optimizerVersion": "v0.20220721",
+            "progLang": "plutus-v2",
+            "repository": "https://github.com/DAppDev/TestDApp/"
+          },
+          "fullScriptHash": "711dcb4e80d7cd0ed384da5375661cb1e074e7feebd73eea236cd68192",
+          "scriptHash": "1dcb4e80d7cd0ed384da5375661cb1e074e7feebd73eea236cd68192",
+          "contractAddress": "addr1wywukn5q6lxsa5uymffh2esuk8s8fel7a0tna63rdntgrysv0f3ms"
+        },
+        {
+          "smartContractInfo": {
+            "era": "basho",
+            "compiler": "plutusTx",
+            "compilerVersion": "1.3.0",
+            "optimizer": "plutonomy",
+            "optimizerVersion": "v0.20220721",
+            "progLang": "plutus-v2",
+            "repository": "https://github.com/DAppDev/TestDApp/"
+          },
+          "fullScriptHash": "384da5375661cb1e07713eea236cd681921dcb4e80d7cd0ed4e7feebd7",
+          "scriptHash": "6cd68191dcb4e80d7c5661cb1e074e7feebd73eea232d0ed384da537",
+          "contractAddress": "addr1wywukn5q6lrdntgrysv0f7a0tna63ymffh23ms5uxsaesuk8s8fel"
+        }
+  ]
+}
+```
+
+The metadata should be discoverable by all certification stakeholders, including end-users, DApp developers, and ecosystem components, such as light wallets and DApp stores. Information should be indexable by certification issuer, DApp developer, DApp and DApp version.
+
+It should be possible for there to be multiple versions of metadata published by the same certification issuer for the same version of a DApp, particularly when a (minor) update of the evidence is necessary. In such a case it should be possible to identify the most recent version of the report for that DApp version. It will also be the case that the same DApp may have certification metadata provided by multiple different certification issuers.
+
+It should be possible for wallets to identify to users the certification status of a DApp when they are signing a transaction that is being submitted to a deployed DApp.
+
+### Custom fields
+
+Certification issuers should be free to add additional fields to fit some additional needs.
+
+### List of Certification issuers
```

Comment:
What's the process for being added to this list? can anyone be added? who verifies them before they are added?

Reply:
Can we apply the same process for adding to the list of Cardano's auditors at CIP 52 here as well? If not, then maybe it is a good time for us to start creating a process to verify or add Cardano's auditors, certification issuers, and testing laboratories to current and future CIPs.

Reply:
Cross-referencing https://github.com/cardano-foundation/CIPs/pull/499#pullrequestreview-1386448536.

Reply:
Also related to https://github.com/cardano-foundation/CIPs/pull/499#discussion_r1192957854 ... will hold off on further related comments because they all appear related to the suggestion (above & below) about turning this certification table into a structured document & process to be added to that document.

Reply:
@Ryun1 @MIxAxIM @RSoulatIOHK - I'm resolving this because it looks like the "list" has been replaced by a process, described in the Specification, for Certification Issuers to publish their own credentials on the chain.  If that's incorrect please respond here. 😎